ui/webui/src/components/AnacondaWizard.jsx

```diff
@@ -144,12 +148,15 @@ export const AnacondaWizard = ({ onAddErrorNotification, title }) => {
     );
 };
 
-const Footer = ({ isFormValid, setStepNotification }) => {
+const Footer = ({ isFormValid, setIsFormValid, setStepNotification }) => {
     const [isInProgress, setIsInProgress] = useState(false);
     const [nextWaitsConfirmation, setNextWaitsConfirmation] = useState(false);
     const [quitWaitsConfirmation, setQuitWaitsConfirmation] = useState(false);
 
     const goToStep = (activeStep, onNext) => {
+        // first reset validation state to default
+        setIsFormValid(true);
```

Comment:
Doesn't this trigger rendering of the active step? I had some issues with missing disks on the Review screen.